src/json/config.schema.json

```diff
@@ -142,6 +142,18 @@
             "title": "Checksum command",
             "examples": ["sha256sum FILENAME"]
           },
+          "checksumActionCommand": {
+            "$id": "#/properties/platforms/items/properties/checksumActionCommand",
+            "$ref": "#/definitions/command",
```

Comment:
the `#id` should be `#/properties/platforms/items/properties/checksumAutoCommand`